src/Tasks/Microsoft.NET.Build.Tasks/ResolvePublishAssemblies.cs

```diff
@@ -90,15 +83,11 @@ protected override void ExecuteCore()
                 TaskItem item = new TaskItem(resolvedAssembly.SourcePath);
                 item.SetMetadata("DestinationSubPath", resolvedAssembly.DestinationSubPath);
                 item.SetMetadata("AssetType", resolvedAssembly.Asset.ToString().ToLower());
+                item.SetMetadata("PackageName", resolvedAssembly.Package.Id.ToString().ToLower());
```

Comment:
It would be nice to use these consts here:  [https://github.com/dotnet/sdk/blob/release/2.0.0/src/Tasks/Microsoft.NET.Build.Tasks/MetadataKeys.cs#L15-L16](https://github.com/dotnet/sdk/blob/release/2.0.0/src/Tasks/Microsoft.NET.Build.Tasks/MetadataKeys.cs#L15-L16) so we ensure we are using consistent metadata names across the board.